phpunit/phpunit/4.7/tests/bootstrap.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+require __DIR__.'/../vendor/autoload.php';
+
+use Symfony\Component\Dotenv\Dotenv;
+
+/*
+ * Environment variables can also be specified in phpunit.xml.dist.
+ * Those variables will override any defined in .env.
+ */
+if (!isset($_SERVER['APP_ENV'])) {
```

Comment:
Just by curiosity, why do we use this condition, also in the console? This disallow from having a generic bash script with a conditional behavior depending of the APP_ENV, and then launch specific commands based on this. It does not make sense in my opinion that `APP_ENV=dev ./bin/console s:r` launches the server without reading the `.env` file while `./bin/console s:r` does. I will have the same issue here as I may set the `APP_ENV` var to `test` before launching the tests (I actually do) ; it won't load the `.env` file consequently...

I would expect that the `.env` file is always loaded (except if `APP_ENV` is defined and set to `prod` so we can prevent misuses of `.env` files), if it exists. The DotEnv component already takes care to avoid overriding already defined variables.